fe/src/main/java/org/apache/doris/analysis/SingleRangePartitionDesc.java

```diff
@@ -87,15 +87,13 @@ public short getReplicationNum() {
         return this.properties;
     }
 
-    public void analyze(int partColNum, Map<String, String> otherProperties) throws AnalysisException {
+    public void analyze(int partColNum, Map<String, String> otherProperties, Short replicationNum) throws AnalysisException {
```

Comment:
I don't think it is a good way to analyze this class with a input replicationNum. It is up to the caller to decide how to use this object.
If we do this through input arguments, there will be too many arguments.

Reply:
ok, I will fix it